streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamPartitionAssignorTest.java

```diff
@@ -99,7 +100,11 @@
     private final TaskId task1 = new TaskId(0, 1);
     private final TaskId task2 = new TaskId(0, 2);
     private final TaskId task3 = new TaskId(0, 3);
-    private String userEndPoint = "localhost:2171";
+    private final String userEndPoint = "localhost:2171";
+    private final StreamPartitionAssignor partitionAssignor = new StreamPartitionAssignor();
+    private final MockClientSupplier mockClientSupplier = new MockClientSupplier();
+    private final TopologyBuilder builder = new TopologyBuilder();
```

Comment:
For my own benefits to learn JUnit: Will this field be re-constructed for each test case? Otherwise its `addSource` function is called in multiple test cases and would that cause an issue?